container-runtime/crun/pkg.yaml

```diff
@@ -34,7 +34,14 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
-
+    sbom:
+      outputPath: /rootfs/usr/share/spdx/crun.spdx.json
```

Comment:
I _think_ it should be under `/rootfs/usr/local` for extensions

Reply:
The way I noticed that is the fact that extensions have limited paths they can "write" to, and `/usr/share` is not included: https://github.com/siderolabs/talos/blob/85d6b919890a1aa9c4f94d5b18861cc617134ff9/pkg/machinery/extensions/extensions.go#L11-L27

I think the way SBOM generation works, it "skips" the validator which should have warned about it

Reply:
@dsseng should have more details on this one, I think it runs after all build steps